packages/connectivity/connectivity/pubspec.yaml

```diff
@@ -17,6 +17,8 @@ dependencies:
   flutter:
     sdk: flutter
   meta: "^1.0.5"
+  connectivity_platform_interface:
+    path: ^1.0.2
```

Comment:
Does this work? I think you need to remove `path`. The version number goes right after the plugin name.

Reply:
hahah you are fast. I notice right away but you beat me to it

Reply:
I was in the review page and it refreshed, thanks for fixing! :P